WcaOnRails/app/controllers/results_submission_controller.rb

```diff
@@ -8,22 +8,21 @@ class ResultsSubmissionController < ApplicationController
 
   def edit
     @competition = competition_from_params
+    @results_submission = ResultsSubmission.new
   end
 
   def submit
```

Comment:
Just a though: as we have a model now, maybe it would be more consistent to have `new` and `create` actions instead?